client/src/components/LandingPageBar/useStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { theme } from '../../themes/theme';
+
+const useStyles = makeStyles(() => ({
+  sitterLoginSignup: {
+    position: 'fixed',
```

Comment:
MUI will actually pass the `theme` as an argument, I would utilize it there as opposed to importing the theme.

Example: `makeStyles((theme: Theme) => ({ . . .}));`